declarations/server/dota-api.d.ts

```diff
@@ -1,3 +1,8 @@
+/// <reference path="dota-enums.d.ts" />
+/// <reference path="dota-gameevents.d.ts" />
+/// <reference path="dota-modifier-properties.d.ts" />
+/// <reference path="dota-std.d.ts" />
+
```

Comment:
These shouldn't be in here.

Reply:
They make it possible to reference dota-api.d.ts with something like
```/// <reference path="../../../../declarations/server/dota-api.d.ts" />```

I have a folder where I download the latest declarations into and reference them like this. Others might want to do the same so I think they should stay.

Reply:
Should be in your code, not in declarations.

Reply:
Ah ok.